packages/theme/src/colors.ts

```diff
@@ -61,3 +61,24 @@ export const colorVariants = {
 } as const;
 
 export type ThemeColorVariant = keyof typeof colorVariants;
+
+export const currencyColors: Record<string, CSSColor> = {
```

Comment:
I would prefer name `coinsColors` because currency it's probalematic because you don't know if it's fiat or crypto currency.

Also please create for it it's own file and also add to `useNativeStyles` so it will be easily accessible anywhere.